src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -607,6 +607,7 @@ static void glb_rr_connectivity_changed(grpc_exec_ctx *exec_ctx, void *arg,
   rr_connectivity_data *rr_conn_data = arg;
   glb_lb_policy *glb_policy = rr_conn_data->glb_policy;
   gpr_mu_lock(&glb_policy->mu);
+  grpc_lb_policy *maybe_unref = NULL;
```

Comment:
Instead of storing a pointer, how about just using `bool unref_needed = false`?  The value of `glb_policy` won't change after unlocking, so there shouldn't be any need to copy the pointer.